package-lock.json

```diff
@@ -0,0 +1,5439 @@
+{
+  "name": "generator-azuresfcsharp",
```

Comment:
This seems like a generated file, is it required to be checked in ? 
Can you remove if not required ?